crates/interpreter/src/evaluate.rs

```diff
@@ -107,6 +107,30 @@ pub fn evaluate(emit: &EmitResult, global: Rc<RefCell<Object>>) -> Result<JSValu
                 stack.push(JSValue::Boolean(strict_equality(&lhs, &rhs)))
             }
 
+            Opcode::Lt => {
+                let rhs = stack.pop().ok_or(EvalError::EmptyStack)?;
+                let lhs = stack.pop().ok_or(EvalError::EmptyStack)?;
+                stack.push(JSValue::Boolean(to_number(&lhs) < to_number(&rhs)))
```

Comment:
Can you add single function for comparison for "<",
with "FIXME" reference to https://tc39.es/ecma262/#sec-abstract-relational-comparison
and use it everywhere?

the comparison is not done by numbers for several cases.

Reply:
So far I really haven't tried to really make the interpreter spec compliant. I think we should probably call that out by adding a general comment somewhere. I am still going to add a FIXME just so if somebody comes across this code they don't just copy and paste it.